README.md

```diff
@@ -176,14 +176,27 @@ or the following:
 1. Go to the root directory of this repository. (Make sure that the **CMock**
    submodule is cloned as described [above](#checkout-cmock-submodule))
 
-1. Run the _cmake_ command: `cmake -S test -B build`
-
-1. Run this command to build the library and unit tests: `make -C build all`
+1. Run the _cmake_ command:
+    ```
+    cmake -S test -B build/ \
+              -G "Unix Makefiles" \
+              -DCMAKE_BUILD_TYPE=Debug \
+              -DBUILD_CLONE_SUBMODULES=ON \
+              -DUNITTEST=1 \
+              -DCMAKE_C_FLAGS='--coverage -Wall -Wextra -Wsign-compare -Werror -DNDEBUG -DLIBRARY_LOG_LEVEL=LOG_DEBUG'
```

Comment:
Does other libraries use this DNDEBUG option as well while building unit tests?

Reply:
kvs-rtp and the cmake command used during the CI for MQTT, both use this flag. Not sue about any other libraries. @aggarg could you please guide. Thanks.

Reply:
Yes, it is used to remove asserts so that coverage calculation is not impacted because of those.